recipes/xontrib-ssh-agent/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "xontrib-ssh-agent" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4fa431e89e9a2a8d2311b0384c92eb4e8e69588e855852c8b2ca78aeb6cbf360
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - xonsh
+
+test:
+  commands:
+    - test -f "${SP_DIR}/xontrib/ssh_agent.xsh"
+
+about:
+  home: https://bitbucket.org/dyuri/xontrib-ssh-agent
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

Comment:
Would be good to ask upstream to ship this in the PyPI package.

Reply:
yep!